packages/alfa-rules/src/common/predicate/is-rendered.ts

```diff
@@ -27,8 +27,20 @@ export function isRendered(
           return false;
         }
 
+        // <option> elements that are descendants of <select> elements will not
```

Comment:
`<optgroup>` under `<select>` seem to be treated the same ay (rendered as label).

Reply:
Good point 👍